app/services/check_answers/section_lister_service.rb

```diff
@@ -58,9 +58,20 @@ def build_field(field_data, label_set, parent_screen)
 
       label = field_data.fetch(:label, field_data.fetch(:attribute))
 
+      disputed = if field_data.key? :disputed_attribute
+                   if field_data.key? :disputed_item
+                     @session_data[field_data[:disputed_attribute]]&.include? field_data[:disputed_item]
+                   else
+                     @session_data[field_data[:disputed_attribute]]
+                   end
+                 else
+                   false
+                 end
+
       Field.new(label: "#{label_set}_fields.#{label}",
                 type: field_data[:type],
                 value:,
+                disputed?: disputed && value.present?,
```

Comment:
When the `value` variable is set to the property `property_owned` attribute, and it is set to `"none"`, the `value.present?` will still be truthy, leading to the potential for:
<img width="675" alt="Screenshot 2022-11-09 at 16 05 19" src="https://user-images.githubusercontent.com/113888736/200880412-9b2277b3-990f-4e42-a9bc-0d35a328d61b.png">